src/gui/DatabaseWidget.cpp

```diff
@@ -797,7 +797,11 @@ void DatabaseWidget::mergeDatabase(bool accepted)
         }
 
         Merger merger(srcDb.data(), m_db.data());
-        merger.merge();
+        bool databaseChanged = merger.merge();
+
+        if (!databaseChanged) {
+            showMessage(tr("Database was not modified by merge operation."), MessageWidget::Information);
+        }
```

Comment:
@AlexJoss at that point we might also want a message if the database was change. The message we have in the CLI is `Successfully merged the database files.`